benchmarks/cosmos-exim/main.go

```diff
@@ -128,7 +128,10 @@ func runExport(dbPath string) (int64, map[string][]*iavl.ExportNode, error) {
 			return 0, nil, err
 		}
 		start := time.Now().UTC()
-		exporter := itree.Export()
+		exporter, err := itree.Export()
+		if err != nil {
+			return 0, nil, err
+		}
```

Comment:
IIRC, this is bit that was causing panic/failure last time. However, see other files/changes holistically.

Reply:
No, it was in `export.go`, as per below.